packages/components/src/unit-control/test/index.js

```diff
@@ -215,7 +215,7 @@ describe( 'UnitControl', () => {
 			expect( input.value ).toBe( '300px' );
 			expect( state ).toBe( 50 );
 
-			user.keyboard( '{Escape}' );
```

Comment:
This change can be dropped after rebasing on top of `trunk` since it was included in #40790